src/main/resources/templates/db_scripts/schema.json

```diff
@@ -698,17 +698,33 @@
          }
        ]
      },
+    {
+      "tableName": "order_template_categories",
+      "fromModuleVersion": "mod-orders-storage-13.9.0",
+      "withMetadata": true,
```

Comment:
lets add this line to enable optimistic locking
 "withOptimisticLocking": "failOnConflict",